packages/dnote-server/build.sh

```diff
@@ -0,0 +1,45 @@
+TERMUX_PKG_HOMEPAGE=https://www.getdnote.com/
+TERMUX_PKG_DESCRIPTION="dnote server"
+TERMUX_PKG_LICENSE="AGPL-V3,GPL-3.0"
+TERMUX_PKG_MAINTAINER="Ravener <ravener.anime@gmail.com>"
+TERMUX_PKG_VERSION=2.0.1
```

Comment:
Need to add `TERMUX_PKG_REVISION=3` since previous version of the package had `TERMUX_PKG_REVISION=2`